### PR TITLE
Fixed fps error when using open source

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: sdk/src/connections/target/adsd3500_sensor.cpp [hunk 1]
```diff
@@ -170,26 +171,25 @@ Adsd3500Sensor::~Adsd3500Sensor() {
         for (unsigned int i = 0; i < dev->nVideoBuffers; i++) {
             if (munmap(dev->videoBuffers[i].start,
                        dev->videoBuffers[i].length) == -1) {
-                LOG(WARNING)
-                    << "munmap error "
-                    << "errno: " << errno << " error: " << strerror(errno);
+                LOG(WARNING) << "munmap error " << "errno: " << errno
```
Comment (dNechita) [inline sdk/src/connections/target/adsd3500_sensor.cpp:174]: There are a lot of changes related to code formatting. Why?

Comment (liealex) [reply sdk/src/connections/target/adsd3500_sensor.cpp:174]: Those changes were made automatically by vscode option "format on save". I will remove them.

**Comment by dNechita** [comment]
Can you please explain the fix? It's not straight forward.
The commit message doesn't help either.

**Comment by liealex** [comment]
When using opensource, m_hostConnectionType has the value "USB". The if condition (https://github.com/analogdevicesinc/ToF/blob/rel-4.3.0/sdk/src/connections/target/adsd3500_sensor.cpp#L243) will not be satisfied and file descriptor will not be initialized.

When opensource is not used, m_hostConnectionType will have a random value "14622345" (here it is not initialized, hence the random value) and the if condition will be satisfied.

To solve those problems, I initialized m_hostConnectionType to "TARGET"

**Comment by rbudai98** [comment]
The same bug was a problem for the ROS2 node as well, it was resolved here here: https://github.com/analogdevicesinc/ToF/pull/498

**Pull request merged**